int64 num            # 유효한 클러스터의 개수
float64[] centroid_x # 유효한 클러스터들의 x 좌표
float64[] centroid_y # 유효한 클러스터들의 y 좌표
float64[] aveD    # 각 클러스터의 중심점 x 좌표 리스트
float64[] minD    # 각 클러스터의 중심점 y 좌표 리스트
float64[] angle   # 각 클러스터의 각도 (필요한 경우)
float64[] size    # 각 클러스터의 크기 (필요한 경우)
